.bazelrc

```diff
@@ -24,26 +24,39 @@ build:cache --experimental_remote_cache_compression
 
 # Flags shared across remote configs
 build:remote-shared --remote_upload_local_results
-build:remote-shared --host_platform=@buildbuddy_toolchain//:platform
-build:remote-shared --platforms=@buildbuddy_toolchain//:platform
-build:remote-shared --crosstool_top=@buildbuddy_toolchain//:toolchain
-build:remote-shared --extra_toolchains=//:sh_toolchain
 build:remote-shared --remote_timeout=600
 build:remote-shared --remote_download_minimal
 build:remote-shared --experimental_repo_remote_exec
 build:remote-shared --jobs=100
 build:remote-shared --verbose_failures
 
+build:remote-shared-linux --config=remote-shared
+build:remote-shared-linux --host_platform=@buildbuddy_toolchain//:platform
+build:remote-shared-linux --platforms=@buildbuddy_toolchain//:platform
+build:remote-shared-linux --crosstool_top=@buildbuddy_toolchain//:toolchain
+build:remote-shared-linux --extra_toolchains=//:sh_toolchain
+
 # Build with --config=remote to use BuildBuddy RBE.
-build:remote --config=remote-shared
+build:remote --config=remote-shared-linux
 build:remote --config=cache
 build:remote --remote_executor=grpcs://remote.buildbuddy.io
 
 # Build with --config=remote-dev to use BuildBuddy RBE.
-build:remote-dev --config=remote-shared
+build:remote-dev --config=remote-shared-linux
 build:remote-dev --config=cache-dev
 build:remote-dev --remote_executor=grpcs://remote.buildbuddy.dev
 
+# Build with --config=remote-mac from a MacOS machine to run builds and tests
+# remotely on Linux executors.
+build:remote-mac --config=remote-shared
+build:remote-mac --config=cache
+build:remote-mac --cpu=k8
+build:remote-mac --host_cpu=k8
+build:remote-mac --host_platform=@buildbuddy_toolchain//:platform_linux
+build:remote-mac --crosstool_top=@buildbuddy_toolchain//:ubuntu_cc_toolchain_suite
+build:remote-mac --extra_toolchains=@buildbuddy_toolchain//:ubuntu_cc_toolchain
```

Comment:
It looks like the only difference between this config and the `remote-shared-linux` config is that it's missing the `--platforms` flag. Does it still work if you add `--platforms` here? If so, I think the config could be simplified a bit since the flags would be the same between both linux and mac.